client/src/bl531_app.py

```diff
@@ -0,0 +1,107 @@
+from dash import Dash
+from layout.app_layout import get_app_layout
+from callback.move import move_callback
+from callback.read import read_callback
+from callback.scaler import plot_scaler
+import dash_bootstrap_components as dbc
+import sys, os, requests
+
+#sys.path.append("/home/bl531/bl531_gui/beamline531_gyl")
+#from beamline_service.epicsDB.epicsdb_utils import getListOphydDashItems
+from model import BeamlineComponents
+from src.epics_db.epicsdb_utils import get_ophyd_dash_items
+
+
+def get_beamline_components(BL_API_URL, BL_API_KEY, BL_UID):
+    # Get beamline PVs from MongoDB as OphydDash object
+    url = f"{BL_API_URL}/beamline/{BL_UID}/components"
+    print(url)
+    response = requests.get(url, headers={"api_key": BL_API_KEY})
+    print(response)
+    if response.status_code != 200:
+        raise BeamlineComponentsNotFound(f"Status code: {response.status_code}")
```

Comment:
Let's also import the BeamlineComponentsNotFound class in this script

Reply:
Created the `BeamlineComponentsNotFound` in `client/src/model.py' and it's imported to `bl531_app.py`